nico/backends/sdl2.nim

```diff
@@ -1829,9 +1829,10 @@ proc getRecordSeconds*(): int =
     return 0
 
 when defined(emscripten):
+  type cbool* {.importc: "bool", nodecl.} = uint8
   type em_callback_func* = proc() {.cdecl.}
   {.push importc.}
-  proc emscripten_set_main_loop*(f: em_callback_func, fps, simulate_infinite_loop: cint)
+  proc emscripten_set_main_loop*(f: em_callback_func, fps: cint, simulate_infinite_loop: cbool)
```

Comment:
```suggestion
  proc emscripten_set_main_loop*(f: em_callback_func, fps: cint, simulate_infinite_loop: bool)
```

Given that cbool and Nim bool both defer to `_Bool` it might be okay to reuse Nim bool here and it might be the reason why cbool doesn't exist in Nim.

I leave that choice to the maintainers of the library.